stack/tc_stack.rb

```diff
@@ -0,0 +1,38 @@
+require "test/unit"
+require_relative "stack"
+
+class TestStack < Test::Unit::TestCase
+
+  def test_add_item_to_stack
+    size= Stack.new(2)
+    item = size.push(6)
+    item = size.push(7)
+    assert_equal(1,item)
+  end
+
+  def test_check_overflow
+    size= Stack.new(1)
+    item = size.push(6)
+    assert_raise RuntimeError do
```

Comment:
Let's move away from this RuntimeError, we send user message. `return "stack is overflow"`